src/page-modules/departures/stop-place/index.tsx

```diff
@@ -0,0 +1,213 @@
+import { MapWithHeader } from '@atb/components/map';
+import { DepartureData } from '..';
+import style from './stop-place.module.css';
+import { MonoIcon } from '@atb/components/icon';
+import {
+  formatLocaleTime,
+  formatToClockOrRelativeMinutes,
+  secondsBetween,
+} from '@atb/utils/date';
+import { PageText, useTranslation } from '@atb/translations';
+import dictionary from '@atb/translations/dictionary';
+import { useTransportationColor } from '@atb/utils/use-transportation-color';
+import { TransportIcon } from '@atb/components/transport-mode';
+import Link from 'next/link';
+import { Typo } from '@atb/components/typography';
+import { useState } from 'react';
+import { andIf } from '@atb/utils/css';
+import { useRouter } from 'next/router';
+import { Departures } from '@atb/translations/pages';
+
+export type StopPlaceProps = {
+  departures: DepartureData;
+};
+export function StopPlace({ departures }: StopPlaceProps) {
+  const router = useRouter();
+  return (
+    <section className={style.stopPlaceContainer}>
+      <div className={style.mapContainer}>
+        <MapWithHeader
+          id={departures.stopPlace.id}
+          name={departures.stopPlace.name}
+          position={[
+            departures.stopPlace.position.lon,
+            departures.stopPlace.position.lat,
+          ]}
+          layer="venue"
+          onSelectStopPlace={(id) => router.push(`/departures/${id}`)}
+        />
+      </div>
+      <div className={style.quaysContainer}>
+        {departures.quays.map((quay) => (
+          <EstimatedCallList key={quay.id} quay={quay} />
+        ))}
+      </div>
+    </section>
+  );
+}
+
+type EstimatedCallListProps = {
+  quay: DepartureData['quays'][0];
+};
+
+export function EstimatedCallList({ quay }: EstimatedCallListProps) {
+  const { t } = useTranslation();
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(false);
+
+  return (
+    <div>
+      <button
+        className={andIf({
+          [style.listHeader]: true,
+          [style['listHeader--collapsed']]: isCollapsed,
+        })}
+        onClick={() => setIsCollapsed(!isCollapsed)}
+        aria-label={
+          isCollapsed
+            ? t(Departures.stopPlace.quaySection.a11yExpand)
+            : t(Departures.stopPlace.quaySection.a11yMinimize)
+        }
+      >
+        <div className={style.flex__row}>
+          <Typo.h3
+            className={style.textColor__secondary}
```

Comment:
Maybe the `Typo` component could have color as a prop too, like the `textType`, but that's for a different PR.

Reply:
I agree! 👍